internal/helm/helm.go

```diff
@@ -96,8 +96,16 @@ func RemoveService(logger logr.Logger, cluster *kubernetes.Cluster, app models.A
 		return errors.Wrap(err, "create a helm client")
 	}
 
-	// err == nil => passed through unchanged
-	return errors.Wrap(client.UninstallReleaseByName(names.ServiceReleaseName(app.Name)), "cleaning up release")
+	// Ignore errors. The release may not be present. For example due to an aborted
+	// deployment. Note that a multitude of different errors was seen for essentially the same
+	// thing, depending on exact timing of deletion to partial creation. Just ignoring a
+	// specific one is fraught. Report, in case we were to generous and debugging is required.
+	if err != nil {
+		logger.Info("release deletion issue", "error", err)
+	}
+
+	_ = client.UninstallReleaseByName(names.ServiceReleaseName(app.Name))
```

Comment:
This should be the other way around

```suggestion
	err := client.UninstallReleaseByName(names.ServiceReleaseName(app.Name))
	if err != nil {
		logger.Info("release deletion issue", "error", err)
	}

```

Reply:
:facepalm:

Reply:
Done. Commit 64a9d9ff58ae235c1ec22c419fd0669292826e7e. Note, `=`, not `:=`.

Reply:
Ups, right!